tool/install_flex_dependencies.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+cd  flex
+npm install
+npm install @inexor-game/connector
```

Comment:
Why would `npm install` not install all necessary dependencies at the first time?